include/crypto/cipher_structs.h

```diff
@@ -103,6 +124,11 @@ struct cipher_ctx {
 		 * available to caller
 		 */
 		void *handle;
+		/* For PRNG operations */
+		struct {
+			u8_t *data;
+			size_t len;
+		} personalization;
```

Comment:
I saw around the repo, two fields, `personalization_data` and `personalization_data_len`. Wouldn't be better keep this pattern ?